public/style.css

```diff
@@ -37,3 +38,64 @@ h2 {
   margin: 5vh 15vw;
   text-align: center;
 }
+
+/* flex stuff   */
+
+figure img{
+  width: 200px;
+}
+.header{
+  border:1px solid black;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  background-color: #E0E0E0;
+}
+.header h3{
+  margin-left: 150px;
+  color: gray;
```

Comment:
It's best to alphabetize css styles. This will help for readability and also for debugging in the future.